packages/web/src/index.tsx

```diff
@@ -12,7 +12,7 @@ const locationChangeHandler = initAnalytics();
 ReactDOM.hydrate(
   <Provider store={store}>
     <BrowserRouter>
-      <App locationChangeHandler={locationChangeHandler} />
+      <App locationChangeHandler={locationChangeHandler} hostname={window.location.host} />
```

Comment:
Why do you need to pass a property here if `location` is available with both `window` or `useLocation` hook inside any component?

Reply:
We can't use `window` in server-rendered components. `useLocation` does not have enough data to make full link.